docs/troubleshooting/existing-policy-cannot-be-added.mdx

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_label: Existing policy can't be added
+---
+
+# Existing policy can't be added
+
+## Problem
+
+When attempting to add an existing policy from Github, I get the following error message:
```

Comment:
I would specify the error code, since algolia can index the text but can't index the error code in the image.